### ISSUE TITLE
Question regarding the use of Val set and Test set in the PreTrained model

### ISSUE THREAD

**Issue by shivammehta25** [NVIDIA/tacotron2#510]
Hello,

Did you use all of the dataset (i.e all Train, Test and Val set) or there was some holdout set in the PreTrained model? 

Regards,
Shivam
